src/doc/en/installation/index.rst

```diff
@@ -80,9 +80,25 @@ Windows
       Then go to the Microsoft Store and install Ubuntu (or another
       Linux distribution). Start Ubuntu from the start menu.
 
-      On the Linux running on WSL, you always have root access, so you
-      can use any of the installation methods described below for
-      Linux.
+      In an Ubuntu (WSL) terminal, type the following commands to install
+      Sage from conda-forge.
+
+      .. code-block:: shell
+
+         $ curl -L -O https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-$(uname)-$(uname -m).sh
```

Comment:
Please use miniforge. Usage of mambaforge is discouraged since a few months.

Reply:
Reference for that please.

Reply:
I don't remember where I read it. Probably in the mircoforge docs or in some issue in their github repo.

Reply:
In the future, when you request a change, please take the moment to back it up with a reference from the beginning. This is gratuitous friction.

The change that you request here is not necessary. https://github.com/conda-forge/miniforge?tab=readme-ov-file#should-i-choose-one-or-another-going-forward-at-the-risk-of-one-of-them-gettting-deprecated is clear: "there are no plans to deprecate Mambaforge"

Reply:
Why? Your link clearly states "if you had to start using one today, we recommend to stick to Miniforge." Why should we not follow this recommendation?

Reply:
We are not "starting". 

The instructions are copied from our existing, tested conda-forge instructions in conda.rst

I don't have objections to changing them, but it's not going to happen in this PR. Scope matters.

Reply:
A user following these instructions would be "starting".

Reply:
That's not an interesting discussion. 

If you are interested in this change, prepare a follow-up PR, change it uniformly, and test it properly.

Reply:
You are adding something suboptimal not me... It's not the task of the reviewer to fix issues introduced in the PR that she is reviewing.